tests/percy/index.js

```diff
@@ -0,0 +1,36 @@
+const PercyScript = require('@percy/script');
+const HttpServer = require('./server');
+const SnapshotDirector = require('./snapshotdirector');
+const StandardPhotographer = require('./standardphotographer');
+const IframePhotographer = require('./iframephotographer');
+
+const PORT = 5042;
+
+PercyScript.run(async (page, percySnapshot) => {
+  const server = new HttpServer({
+    dir: 'test-site/public',
+    port: PORT
+  })
+
+  server.start();
+
+  const standardPhotographer = new StandardPhotographer({
+    page: page,
+    percySnapshot: percySnapshot,
+    siteUrl: `http://localhost:${PORT}`,
+  });
+
+  const iframePhotographer = new IframePhotographer({
+    page: page,
+    percySnapshot: percySnapshot,
+    siteUrl: `http://localhost:${PORT}`,
+    iframePage: 'iframe_test'
+  });
+
+  const snapshotDirector = new SnapshotDirector(standardPhotographer);
```

Comment:
this refactor is super cool!

Reply:
Ya, I had fun with this one, and I'm hoping that it will make it easy to write code for new snapshots